requirements/edx/github.txt

```diff
@@ -4,6 +4,42 @@
 #   * @mollydb to check licensing
 #   * One of @e0d, @feanil, @fredsmith, @maxrothman, or @jibsheet
 #     to check system requirements
+#
+# A correct GitHub reference looks like this:
+#
+#   git+https://github.com/OWNER/REPO-NAME.git@TAG-OR-SHA#egg=KIT-NAME==VERSION
+#
+# For example:
+#
+#   git+https://github.com/edx/edx-lint.git@v0.3.2#egg=edx_lint==0.3.2
+#
+# where:
+#
+#   OWNER = edx
+#   REPO-NAME = edx-lint
+#   TAG-OR-SHA = v0.3.2
+#   KIT-NAME = edx_lint
+#   VERSION = 0.3.2
+#
+#
+# Rules to follow:
+#
+#   * Don't leave out any of these pieces.
+#
+#   * Don't use -e
```

Comment:
This rule is broken nearly 30 times in this very file.  This bears some explanation.